src/assets/styles.scss

```
@@ -0,0 +1,7 @@
+@font-face {
+	font-family: 'myfont';
```

Small note - good practice to have consistent naming.

```suggestion
	font-family: 'lato-regular';
```

---

Updated this